Dockerfile

```diff
@@ -27,8 +27,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     && rm -rf /var/lib/apt/lists/* \
     && ln -sf /dev/stdout /var/log/nginx/access.log \
     && ln -sf /dev/stderr /var/log/nginx/error.log \
-    && npm install -g npm@7 \
-    && npm config set maxsockets 5
```

Comment:
curious why we don't need this anymore?

Reply:
this issue has been addressed in a newer version of npm https://github.com/npm/cli/issues/2977#issuecomment-809704961